src/Core/Components/InputFile/FluentInputFile.razor.cs

```diff
@@ -12,7 +12,7 @@ public partial class FluentInputFile : FluentComponentBase, IAsyncDisposable
     private ElementReference? _containerElement;
     private InputFile? _inputFile;
     private IJSObjectReference? _containerInstance;
-
+    
```

Comment:
Can you remove these new spaces ?

Reply:
I can't since there is no space. The space got removed and added back. The file looks the same as the one in this repository.
![grafik](https://github.com/user-attachments/assets/30c9f34d-cc5c-48e1-a670-3855740c407a)